sonar-text-dotnet/src/SonarLint.Secrets.DotNet.UnitTests/EntropyCheckerTest.cs

```diff
@@ -16,7 +16,7 @@ namespace SonarLint.Secrets.DotNet.UnitTests
     [TestClass]
     public class EntropyCheckerTest
     {
-        private readonly string commonTestFilePath = "..\\sonar-text-plugin\\src\\test\\resources\\EntropyChecker\\";
+        private const string commonTestFilePath = "..\\sonar-text-plugin\\src\\test\\resources\\EntropyChecker\\";
```

Comment:
I think the constants should start with uppercase.

Reply:
This has been replaced by the universal const in the new class with consts, which has the correct case. :D